src/services/chains/index.ts

```
@@ -26,53 +28,75 @@ export interface TransactionDetailInput {
 
-export function validateTransactionWithInputData(
+export async function validateTransactionWithInputData(
   transaction: NetworkTransactionInfo,
   input: TransactionDetailInput,
-): never | void {
-  if (transaction.to.toLowerCase() !== input.toAddress.toLowerCase()) {
-    throw new Error(
-      i18n.__(
-        translationErrorMessagesKeys.TRANSACTION_TO_ADDRESS_IS_DIFFERENT_FROM_SENT_TO_ADDRESS,
-      ),
+): Promise<never | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)